lib/Dialect/Comb/CombOps.cpp

```diff
@@ -181,22 +181,21 @@ LogicalResult ReplicateOp::verify() {
 // Variadic operations
 //===----------------------------------------------------------------------===//
 
-static LogicalResult verifyUTVariadicOp(Operation *op) {
+static LogicalResult verifyUTBinOp(Operation *op) {
   if (op->getOperands().empty())
     return op->emitOpError("requires 1 or more args");
-
   return success();
 }
```

Comment:
Mega nit: this appears to be just a name-change as opposed to anything functional? You can consider landing this separately. This is not a blocker.